tests/e2e/common/kubernetes.go

```diff
@@ -213,3 +215,18 @@ func cleanupNodeFeatureRules(ctx context.Context, cli *nfdclient.Clientset) {
 		}
 	}
 }
+
+// CreateKubernetesClient creates a Kubernetes clientset from the given kubeconfig file
+func CreateKubernetesClient(kubeconfig string) (*kubernetes.Clientset, error) {
```

Comment:
Given that our tests already check kubernetes objects using a client, I'm surprised to see this function added **without** additional modifications to the existing code. Can we reuse what we already have to create a client?